gammapy/data/data_store.py

```diff
@@ -14,12 +14,17 @@
 
 __all__ = ["DataStore"]
 
+ALL_IRFS = ["aeff", "edisp", "psf", "bkg", "rad_max"]
+ALL_HDUS = ["events", "gti"] + ALL_IRFS
 REQUIRED_IRFS = {
-    "full-enclosure": ["aeff", "edisp", "psf", "bkg"],
-    "point-like": ["aeff", "edisp"],
-    "all-optional": ["aeff", "edisp", "psf", "bkg", "rad_max"],
+    "full-enclosure": {"aeff", "edisp", "psf", "bkg"},
+    "point-like": {"aeff", "edisp"},
+    "all-optional": {},
```

Comment:
maybe we can just delete this option now

Reply:
That would be an API breaking change